universal/router/matches/index.js

```diff
@@ -0,0 +1,13 @@
+/* @flow */
+import WithSubRoutes from './WithSubRoutes'
+import { Redirect, Miss, Match, NavigationPrompt } from 'react-router'
+
+const match = {
+  Match,
+  Miss,
+  NavigationPrompt,
+  WithSubRoutes,
+  Redirect
+}
+
+export default match
```

Comment:
Also, if we updated this to 
```javascript
export WithSubRoutes from './WithSubRoutes'
export {  Redirect, Miss, Match, NavigationPrompt } from 'react-router'

// we could use it as follows
import * as match from '../matches_wherever_they_are'

// or

import { Redirect, Miss, Match, NavigationPrompt } from '../matches_wherever_they_are'
```

giving the user the flexibility for both a namespaced import or the destructured approach. 

Thoughts?